### DISCUSSION TITLE
Color picker

### DISCUSSION THREAD

**Discussion by senyai** [microsoft/vscode-discussions#560] [Extension Development QnA] [answered]
Hi! I need to associate a color with an scm branch and for that I want to show a color picker. I expected there to be a built-in command, but I couldn't find one. Does anyone have an idea on how to ask a user to input a color?

**Comment by isidorn** [comment]
Maybe this could be triggered by a command. @aiday-mar might know

**Comment by aiday-mar** [comment]
HI @senyai, there is the command `Show or Focus Standalone Color Picker` which allows one to select a color and add it to the editor. But this is probably of no use to you. When you mention an scm branch, are you using GitHub scm, or are you developing some sort of custom scm?

**Comment by senyai** [comment]
Hi @aiday-mar, I develop fossil-scm extension where a user can assign a color to a branch and it looks [like this](https://fossil-scm.org/home/timeline). I wouldn't say color-picker is essential for my extension, but it would be nice. And I imagine someone can create an image editor for vscode where color picker would be useful.

**Comment by aiday-mar** [comment, marked as answer]
I see thanks for the clarification, we currently do not have such a feature in VS Code. I have added an issue related to this here: https://github.com/microsoft/vscode/issues/187473 which you can track.

**Reply by burkeholland** [reply]
@senyai Marking this as the answer. If you feel that this is a mistake, please feel free to remove the answer designation.
